core/src/test/scala/unit/kafka/server/ControllerMutationQuotaTest.scala

```diff
@@ -139,20 +143,23 @@ class ControllerMutationQuotaTest extends BaseRequestTest {
       val (_, errors) = createTopics(Map("topic" -> 1), StrictDeleteTopicsRequestVersion)
       assertEquals(Set(Errors.NONE), errors.values.toSet)
 
-      // Metric must be there with the correct config
-      waitQuotaMetric(principal.getName, ControllerMutationRate)
+      if (!isKRaftTest()) {
```

Comment:
Why are we only doing these checks in ZK mode? Isn't the controller mutation quota also working in KRaft mode?